internal/storage/web_test_util.go

```diff
@@ -0,0 +1,237 @@
+// Copyright (c) 2022 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0
+//
+// SPDX-License-Identifier: EPL-2.0
+
+package storage
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"net/http"
+	"net/url"
+	"runtime"
+	"strconv"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/eclipse-kanto/software-update/hawkbit"
+)
+
+// Web encapsulates a test HTTP(S) server and testing data
+type Web struct {
```

Comment:
Web (file name, struct etc.) doesn't sound correct. Isn't it a test HTTP server?